distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -154,6 +154,11 @@ grant codeBase "file:/catalog-transformer-csw-profile/persistence-core-impl/pers
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}-", "read";
 }
 
+grant codeBase "file:/solr-factory-impl" {
```

Comment:
❓ Does that mean that because it is no longer embedded in other bundles, we could remove the permissions we were forced to give to the other bundles?